packages/web/.eslintrc.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": ["next/core-web-vitals", "prettier"],
   "plugins": ["unicorn", "simple-import-sort", "unused-imports", "import"],
+  "ignorePatterns": ["/e2e/keplr-extension"],
```

Comment:
NIT:
Thoughts on more closely matching code base convention of placing all tests in "__ tests __" folders (added spaces to avoid italics)?

So, this folder would then be "__ tests_e2e __". 

You can see such a folder in server package.

Also, I think it's a bit more descriptive of what is contained within that folder

Reply:
Let me do it in a separate MR. Thanks for the advise.